### ISSUE TITLE
Add new STARKVIND Air purifier

### ISSUE THREAD

**Issue by derherrgraf** [AlCalzone/ioBroker.tradfri#691]
I noticed that the new air purifier are not yet supported. See the log:
```
2021-11-12 14:32:19.537 - warn: tradfri.0 (1831) found unknown property with key 15025
--
2021-11-12 14:32:19.538 - warn: tradfri.0 (1831) object was: {"3":{"0":"IKEA of Sweden","1":"STARKVIND Air purifier","2":"","3":"1.0.033","6":1,"7":4364},"5750":10,"9001":"NAME_OF_THE_DEVICE","9002":1636224627,"9003":65560,"9019":1,"9020":1636720780,"9054":0,"15025":[{"5900":1,"5902":4176,"5903":0,"5904":259200,"5905":0,"5906":0,"5907":59,"5908":15,"5909":4132,"5910":255024,"9003":0}]}
2021-11-12 14:32:19.540 - warn: tradfri.0 (1831) Unknown accessory type 10. Please send this info to the developer with a short description of the device!
```
I tried to add them, but i don't quite understand how the code works. I found this code/commit as a good reference https://github.com/home-assistant-libs/pytradfri/commit/1a8545e47d12d831e69dced1c0c9b05cbb1f30b5

Please ask if there are questions, or I can help testing stuff :)

**Comment by Kev0123** [comment]
+1 here.

Would like so see this implementation!

**Comment by AlCalzone** [comment]
fixed in v3.1.0. Testing thread is here: https://forum.iobroker.net/topic/50451/test-adapter-tradfri-v3-1-x
